src/assets/locales/fr.json

```diff
@@ -337,23 +382,28 @@
   "preferences-screen.material-quality-setting.medium": "Medium",
   "preferences-screen.preference.allow-multiple-hubs-instances": "Désactiver la sortie automatique quand de multiples instances de hubs sont ouvertes",
   "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoints",
+  "preferences-screen.preference.audio-clipping-threshold": "Seuil pour l'audio clipping",
   "preferences-screen.preference.disable-auto-gain-control": "Désactiver le contrôle automatique du gain du micro",
   "preferences-screen.preference.disable-auto-pixel-ratio": "Désactiver les ajustements automatiques du ratio de pixels",
   "preferences-screen.preference.disable-backwards-movement": "Désactiver les mouvements en marche arrière",
   "preferences-screen.preference.disable-echo-cancellation": "Désactiver l'annulation d'écho du micro",
   "preferences-screen.preference.disable-idle-detection": "Désactiver la sortie automatique si inactif ou en arrière-plan",
   "preferences-screen.preference.disable-movement": "Désactiver le mouvement",
   "preferences-screen.preference.disable-noise-suppression": "Désactiver la suppression de bruit du micro",
+  "preferences-screen.preference.disable-panning": "Désactiver le panning audio gauche/droite",
```

Comment:
"preferences-screen.preference.disable-panning": "Disable audio left/right panning",

Reply:
Is it a panel or an audio effect?

Reply:
audio effect, just an option

Reply:
```suggestion
  "preferences-screen.preference.disable-panning": "Désactiver l'audio panoramique (gauche/droite)"
```

Reply:
Thanks, you're right, "panoramique" is the correct word. See French doc https://developer.mozilla.org/fr/docs/Web/API/Web_Audio_API/Web_audio_spatialization_basics